sn_cli/src/subcommands/files/chunk_manager.rs

```diff
@@ -146,9 +150,15 @@ impl ChunkManager {
             debug!(
                 "All files_to_chunk ({total_files:?}) were resumed. Returning the resumed chunks.",
             );
+            println!("Resumed all {total_files:?} files.");
```

Comment:
These reads oddly to me. `resumed chunking`?  Im not really clear what a resumed chunk is vs a normal chunk? 

we resume the process of chunking, not the chunks themselves? preexisting chuinks were reused?

Reply:
Or are is the chunking of files being resumed?

Reply:
Remove the commit as we currently have clearer prints on the upper layer.